v3/wrap/docs/readme.md

```diff
@@ -0,0 +1,46 @@
+# The UniV3 Wrap
+
+[Uniswap V3](https://docs.uniswap.org/concepts/uniswap-protocol) is a peer-to-peer system designed for exchanging cryptocurrencies on the Ethereum blockchain. It has become one of the most used & trusted decentralized exchanges on the blockchain ([usage statistics](https://defillama.com/protocol/uniswap)).
+
+## Integrate
+
+### Step 1: Polywrap Client
+
+In order to integrate the UniV3 wrap into your applications, the first thing you'll need is a Polywrap client. Currently Polywrap has clients available in:
+- JavaScript / TypeScript
+- Python
+- Rust
+- Go
```

Comment:
I don't think the Go client is in a ready state. Does it have an ethers provider plugin?

Reply:
This was mainly taken from the README in the UniV3 demo, so I didn't use my brain too much when setting it up.

I'll have it removed.